modules/mlops/kubeflow-platform/deployspec.yaml

```diff
@@ -0,0 +1,90 @@
+deploy:
+  phases:
+    install:
+      commands:
+      - bash install_build.sh
+    build:
+      commands:
+      # Export all env params specific to the deployment stuff
+      - export ROOT_DIR=$(pwd)
+      - export CLUSTER_NAME=${ADDF_PARAMETER_EKS_CLUSTER_NAME}
+      - export CLUSTER_REGION=${AWS_DEFAULT_REGION}
+      - export INSTALLATION_OPTION=${ADDF_PARAMETER_INSTALLATION_OPTION} #helm
```

Comment:
in line 12-15, 
it is probably nice to get rid of the comments such as `#helm` etc

Reply:
ok